### PR TITLE
Adding my hometown file for Prague, CZ

### LINKED ISSUE #2: GitHubTeacher's Hometown
## Fuquay Varina, NC
- [x] Create branch
- [x] Add my file
- [x] Commit my file
- [x] Open a pull request
- [ ] Get :+1:
- [ ] Merge pull request

### LINKED ISSUE #7: JiriCermak's Hometown
## Prague, Czech Republic
- [x] Create branch
- [x] Add my file
- [x] Commit my file
- [x] Open a pull request
- [ ] Get :+1: 
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by ghost**
This pull request closes #7

### CODE REVIEW

File: places/prague.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Prague, CZ
+
+- Located in the heart of Europe.
```
Comment (ghost) [inline places/prague.md:3]: Not the literal heart of Europe.

File: places/prague.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- You can eat anywhere :) It is relatively cheap.
```
Comment (k-vaughan) [inline places/prague.md:4]: I'm going to Prague in April. I'll need some more specific recommendations, @jiricermak.

Comment (githubteacher) [inline places/prague.md:4]: If it is my first time in Prague, what should I definitely make time to see? @jiricermak

**Comment by githubteacher** [comment]
:+1: this looks good to :ship:

**Pull request merged**